layouts/partials/nav.html

```diff
@@ -35,13 +35,15 @@
       </li>
 
       {{ if $isHere }}
```

Comment:
Can we add an entry next to "Scalers" in the main navigation?

Something like "Authentication Providers" or so. Thoughts on a better name @zroubalik @JorTurFer ?

Reply:
Something to consider: the Keda.sh top nav is getting crowded. It is already overflowing as you can see from this screenshot:

> <img width="1044" alt="image" src="https://user-images.githubusercontent.com/4140793/182853343-1e23728e-a2c2-4882-9bdb-45c5b464f311.png">

Notice how the top nav is clipped on the right side of the screen.

Reply:
That's OK; I'll create a seperate item as we can move Merch in Project

Reply:
https://github.com/kedacore/keda-docs/issues/872